sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Resources.resx

```diff
@@ -237,4 +237,28 @@
   <data name="CannotCreateCheckpointForEmptyEvent" xml:space="preserve">
     <value>A checkpoint cannot be created or updated using an empty event.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="HandlerHasAlreadyBeenAssigned" xml:space="preserve">
+    <value>Another handler has already been assigned to this event.</value>
+  </data>
+  <data name="HandlerHasNotBeenAssigned" xml:space="preserve">
+    <value>Handler has not been previously assigned to this event.</value>
```

Comment:
You may want to add "This" as a prefix to the sentence.   This particular case is basically "You can't remove the handler you're asking for because it's not the active handler that was assigned."